Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:alpine AS build
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN yarn && \
+    yarn build
+
+VOLUME [ "/usr/src/app/database" ]
```

Comment:
Nitpick: would it be possible to use a parameterized path rather than hard coding it here?

Reply:
The correct place for this would likely best be the commander config options so the path was specifiable there. The volume in the dockerfile is hardcoded.